app/src/test/java/com/example/marvelapp/mapper/EntityCharacterMapperTest.kt

```diff
@@ -15,17 +15,18 @@ import org.mockito.junit.MockitoJUnitRunner
 @RunWith(MockitoJUnitRunner::class)
 class EntityCharacterMapperTest {
 
-    private lateinit var characterEntity: CharacterEntity
-    private lateinit var characterEntityList: List<CharacterEntity>
+    private lateinit var characterEntity: com.example.data.database.entity.CharacterEntity
```

Comment:
import the CharacterEntity instead of put here all the path

Reply:
done